designs/UI/css/business.css

```diff
@@ -0,0 +1,28 @@
+#results{
+    width: 90%;
+    margin: auto;
```

Comment:
What do you think about making margin DRY. I'm not sure, but try to move margin property to a new class/id selector and the nest it with other selectors that neede the property

Reply:
DRY Noted. Will check and group all common props